README.md

```diff
@@ -1,107 +1,122 @@
-# Phrase In-Context Editor for Ruby #
+# phraseapp-in-context-editor-ruby
 
-*This Ruby gem is for use with Ruby (Rails, Sinatra) applications only. Check out the [documentation](https://help.phrase.com/help/translate-directly-on-your-website) to learn how to set up the In-Context Editor with other technologies.*
+![Build status](https://github.com/phrase/phraseapp-in-context-editor-ruby/workflows/Test/badge.svg)
 
-![Build Status](https://github.com/phrase/phraseapp-in-context-editor-ruby/workflows/Test/badge.svg)
+**phraseapp-in-context-editor-ruby** is the official library for integrating [Phrase Strings In-Context Editor](https://support.phrase.com/hc/en-us/articles/5784095916188-In-Context-Editor-Strings) with [i18n](https://github.com/ruby-i18n/i18n) in your React application.
```

Comment:
Bad copy/paste? 😁  It should say `Ruby (Rails, Sinatra)` application, instead of `React` at the end of the sentence.